package.json

```diff
@@ -169,7 +169,7 @@
   },
   "dependencies": {
     "@brave/brave-ui": "0.40.6",
-    "@brave/leo": "github:brave/leo#58a07d5a8f3fea5aff6327d64ad0b58657111c29",
+    "@brave/leo": "github:brave/leo#events-magic",
```

Comment:
This is pointing at a branch at the moment because upstream Nala doesn't support `EventHandlerObject`s in `addEventListener`.

This should be fixed soon :smile: